TripPlanner - Project/TripPlannerFrontend/src/components/Typography/Typography.tsx

```diff
@@ -20,10 +20,44 @@ const Subheading = styled(TextBase)`
   font-size: 2.074rem;
 `;
 
+const Nav = styled.ul`
+  width: 50%;
+  display: flex;
+  list-style-type: none;
+  justify-content: flex-end;
+  font-family: Work Sans, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 15px;
+  color: #919191;
+  & > li {
+    padding-right: 1rem;
+  }
+`;
+
+const NavBar = styled.div`
+  display: flex;
+  align-items: center;
+  width: 100%;
+  justify-content: space-between;
+  & > h3 {
+    padding-left: 1rem;
+    font-family: Work Sans, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 17.4703px;
+    line-height: 20px;
+    letter-spacing: -0.03em;
+  }
+`;
+
```

Comment:
These components are not part of typography. Could you extract them to separate directories under `src/components`?

Also please do not style styled-components `sass`-style. If you have to refer to other component, please make yourself familiar with this: https://styled-components.com/docs/advanced#referring-to-other-components